src/coreclr/inc/holder.h

```diff
@@ -909,8 +909,10 @@ FORCEINLINE void DoTheRelease(TYPE *value)
 template<typename _TYPE>
 using DoNothingHolder = SpecializedWrapper<_TYPE, DoNothing<_TYPE*>>;
 
+#ifndef SOS_INCLUDE
```

Comment:
Why exclude this from SOS?

Reply:
Because SOS has it's own ReleaseHolder that conflicts with this one and it is different enough that it isn't worth changing all the SOS code to use this one.